### PR TITLE
created the new merchant page

### PR DESCRIPTION
**Pull request by ufas-001**
#### What does this PR do?
<Brief Message Usually Ticket Title>
created the new merchant page
#### Description of the task to be completed?
- created the page with route /merchant
- point 2
- point 3

#### How can this manually be tested?
- clone this repo
```
$ git clone https://github.com/bukkaApp/Mybukka-Frontend
```
- install all dependencies
```
$ npm i
```
- start Application
```
$ npm start:dev
```
- log on to _localhost:PORT_ to interact with created page

#### Relevant PT story(ies)
<PT_STORY_ID>

#### Questions
N/A

### CODE REVIEW

File: src/features/merchantPage/common/customButton.jsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import './CustomButton.scss';
+import '../index.scss'
```
Comment (akolliy1) [inline src/features/merchantPage/common/customButton.jsx]: Are sure of this file `import './CustomButton.scss';` should be small letter `c`. And why do you have index styling imported here

File: src/features/merchantPage/common/customButton.jsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+const CustomButton = ({ buttonChildren, spanChildren }) => (
+    <div>
+        <button className="custom-button mt-3">{buttonChildren}</button> <span className="custom-span">{spanChildren}</span>
+    </div>
+);
```
Comment (akolliy1) [inline src/features/merchantPage/common/customButton.jsx]: 2 spaces indentation

File: src/features/merchantPage/common/customButton.scss [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    height: 51px;
+    border: 1px solid #F5E7D8;
+    font-family: Roboto;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 20px;
+    line-height: 23px;
+    text-align: center;
+    color: #9D1621;
+}
```
Comment (akolliy1) [inline src/features/merchantPage/common/customButton.scss:23]: 2 spaces indentation

File: src/features/merchantPage/component/banner.jsx [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+            </div>
+            <div className="content">
+                <h4>Grow Your Takeout Sales by 200%</h4>
+                <p>With the top 3rd-party delivery platform for customer spend and retention.</p>
+                <div className="text-align-center">
+                    <button className="get-started-btn">Get started</button>
+                </div>
+            </div>
+        </div>
+    </div>
```
Comment (akolliy1) [inline src/features/merchantPage/component/banner.jsx]: 2 spaces indentation

File: src/features/merchantPage/component/banner.scss [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+    width: 100%;
+    height: 600px;
+    top: 0px;
+}
+
+.info-content {
+    width: 90%;
+    display: flex;
+    justify-content: space-between;
+    padding-top: 30px;
```
Comment (akolliy1) [inline src/features/merchantPage/component/banner.scss]: 2 spaces indentation

File: src/features/merchantPage/component/banner.scss [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+    text-transform: uppercase;
+    font-weight: normal;
+    font-size: 15px;
+    color: #ffff;
+    font-family: 'Roboto', sans-serif;
+    cursor: pointer;
+    line-height: 18px;
+    margin-top: 30px;
+    margin-bottom: 20px;
+}
```
Comment (akolliy1) [inline src/features/merchantPage/component/banner.scss]: 2 spaces indentation

File: src/features/merchantPage/component/description.css [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+    font-family: Roboto;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 18px;
+    line-height: 24px;
+    text-align: center;
+    color: #949494;
+    width: 220px;
+    margin: auto;
+}
```
Comment (akolliy1) [inline src/features/merchantPage/component/description.css]: 2 spaces indentation

File: src/features/merchantPage/component/footer.jsx [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+                <p>©2019 MYBUKKA INC TERMS PRIVACY </p>
+                <div>
+                    <span class="pr-4">Terms and Policy</span>
+                    <span>Privacy</span>
+                </div>
+            </div>
+
+        </div>
+    </div>
+);
```
Comment (akolliy1) [inline src/features/merchantPage/component/footer.jsx]: 2 spaces indentation

File: src/features/merchantPage/component/navDescription.css [hunk 9]
```diff
@@ -0,0 +1,29 @@
...
+
+.navigation-info p {
+    font-family: Roboto;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 18px;
+    line-height: 24px;
+    width: 390px;
+    color: #ffff;
+}
```
Comment (akolliy1) [inline src/features/merchantPage/component/navDescription.css]: 2 spaces indentation

File: src/features/merchantPage/component/navDescription.jsx [hunk 10]
```diff
@@ -0,0 +1,24 @@
...
+                    alt="map" className="map-img" />
+                </div>
+                <div className="col-md-6 mt-5 pl-3">
+                    <h4 className="mt-5">Connect with new local customers.</h4>
+                    <p> Grow your sales and become a favorite by getting in front of more people at the exact moment they’re ready to buy. They come to us, we introduce them to you.</p>
+                    <CustomButton buttonChildren= '5,000 +' spanChildren= 'daily searches' />
+                </div>
+            </div>
+        </div>
+    </div>
```
Comment (akolliy1) [inline src/features/merchantPage/component/navDescription.jsx]: 2 spaces indentation

File: src/features/merchantPage/component/orderDescription.css [hunk 11]
```diff
@@ -0,0 +1,38 @@
...
+
+.order-info p {
+    font-family: Roboto;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 18px;
+    line-height: 24px;
+    width: 390px;
+    color: #949494;
+}
```
Comment (akolliy1) [inline src/features/merchantPage/component/orderDescription.css]: 2 spaces indentation

File: src/features/merchantPage/component/presenceDesccription.jsx [hunk 12]
```diff
@@ -0,0 +1,25 @@
...
+                    <CustomButton buttonChildren= '10,000 +' spanChildren= 'business online' />
+                </div>
+                <div className="col-md-6 mt-5 pl-3">
+                    <img src="https://res.cloudinary.com/mybukka/image/upload/v1586035063/phone_xiqrcw.jpg" 
+                    alt="meal-image" className="meal-img" />
+
+                </div>
+            </div>
+        </div>
+    </div>
```
Comment (akolliy1) [inline src/features/merchantPage/component/presenceDesccription.jsx]: 2 spaces indentation

**Review by akolliy1 — CHANGES_REQUESTED** [review]
Great, You just need to make a few changes and proper indentation.

**Comment by ufas-001** [comment]
I don't think 2 space indentation is achievable for css and scss. Waiting
for your feedback before I push

**Pull request closed without merging**